### ISSUE TITLE
Showing version of VNM

### ISSUE THREAD

**Issue by civier** [NeuroDesk/neurodesk#56]
It will be good to show somewhere in the GUI the version of the VNM the user is using. Will be very useful for debugging.
Maybe we should have in the menu an "about" window that display credits (our names, our institution names) and version.

**Comment by stebo85** [comment]
yes, good idea. Should be easy to implement. Give it a go and submit a pull request :)

**Comment by civier** [comment]
Hi Steffen,

I would be more than happy to do some pull requests, but I'm up to my neck
at the moment (we're starting to scan again). Just give me time.

Best,
Oren

**Comment by stebo85** [comment]
Dear @civier 

Did you make any progress in implementing this feature you requested?

Kind regards
Steffen
